pkg/php/php.go

```diff
@@ -41,6 +42,8 @@ const (
 	dependencyHashKey = "dependency_hash"
 
 	composerVersionKey = "php"
+	// composerArgsEnv is an environment variable used to pass custom composer variables.
+	composerArgsEnv = "COMPOSER_ARGS"
```

Comment:
Could you please update the name to "GOOGLE_COMPOSER_ARGS" to follow the env var convention.

Reply:
thanks.